easybuild/easyconfigs/s/Sambamba/Sambamba-0.8.0-GCC-10.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'MakeCp'
+
+name = 'Sambamba'
+version = '0.8.0'
```

Comment:
`'commit': 'f898046'` corresponds to `version = '0.8.2'`. This should be updated and the filename changed to `Sambamba-0.8.2-GCC-10.3.0.eb`